packages/lib-react-components/src/helpers/withLayer/withLayer.js

```diff
@@ -41,7 +41,10 @@ function withLayer (WrappedComponent) {
     closeFn: PropTypes.func,
     modal: PropTypes.bool,
     plain: PropTypes.bool,
-    position: PropTypes.string
+    position: PropTypes.string,
+    target: PropTypes.shape({
+      current: PropTypes.node
```

Comment:
The Grommet Layer docs are wrong. This is a DOM node, and is expected to have a `getBoundingClientRect` method (in the Layer code.)